e2e_tests/tests/command/test_run.py

```diff
@@ -350,3 +358,51 @@ def test_killed_pending_command_terminates() -> None:
     else:
         state = cmd.get_command(command.task_id)["state"]
         raise AssertionError(f"Task was in state {state} rather than TERMINATED")
+
+
+@pytest.mark.e2e_gpu  # type: ignore
+def test_k8_mount(using_k8s: bool) -> None:
+    if not using_k8s:
+        pytest.skip("only need to run test on kubernetes")
+
+    mount_path = "/ci/"
+
+    with pytest.raises(subprocess.CalledProcessError):
+        _run_and_verify_failure(
```

Comment:
blocking: is there any other way to make this check more precise, not just checking "was terminated: container failed with non-zero exit code:"?

Reply:
done